### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by gronke**
- Mention talks and tutorials about libiocage at BSD conferences
- Update install section
- Include CLI usage information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,20 +6,20 @@ iocage is a jail/container manager fusioning some of the best features and techn
...
 
-### Project Status
-As of November 2017 this project is *working towards an alpha release*. This means stabilization of command-line and library interfaces, so that proper integration tests can be built. This phase requires manual verification and testing until reaching feature-completion and compatibility with Python [iocage](https://github.com/iocage/iocage) and prior iocage_legacy versions with [ZFS property](https://github.com/iocage/iocage_legacy/tree/master) and [UCL file](https://github.com/iocage/iocage_legacy) config storage.
+### Latest News (August 7th, 2018)
+libiocage is making small but continuous steps to stabilize the interfaces and become used in [iocage/iocage](https://github.com/iocage/iocage). The project was first presented in the talk "[Imprisoning software with libiocage](https://www.bsdcan.org/2018/schedule/events/957.en.html)" at BSDCan 2018 (Video Recording on [YouTube](https://www.youtube.com/watch?v=CTGc3zYToh0)). There will be a Tutorial about [Advanced container management with libiocage](https://2018.eurobsdcon.org/tutorial-speakers/#StefanGronke) on Semtember 20th, 2018 at EuroBSDCon in Bucharest.
```
Comment (skarekrow) [inline README.md]: s/Semtember/September

File: README.md [hunk 2]
```diff
@@ -6,20 +6,20 @@ iocage is a jail/container manager fusioning some of the best features and techn
...
 
-## Install
+Ongoing preparations at this repository and iocage ensure that the transition to using libiocage under the hood of iocage go as smooth as possible for users. Features that exist in iocage will be further improved and tested or announced to be replaced or deprecated shortly. Rumors at iX Systems let imagine that libiocage once finds its way into FreeNAS where it can play its full strength behind a Web GUI.
```
Comment (skarekrow) [inline README.md]: Just iXsystems and "let one imagine that libiocage may find its way into"

**Comment by gronke** [comment]
@skarekrow thanks for the review. Your requested changes are addressed.

**Comment by skarekrow** [comment]
@gronke I still see iXsystems spaced in the commit?

**Comment by gronke** [comment]
Oops. @skarekrow it's updated now.

File: README.md [hunk 3]
```diff
@@ -63,6 +63,46 @@ jail.create("11.1-RELEASE")
...
+  rename      Rename a stopped jail.
+  restart     Restarts the specified jails.
+  set         Sets the specified property.
+  snapshot    Take and manage resource snapshots.
+  start       Starts the specified jails or ALL.
+  stop        Stops the specified jails or ALL.
+  update      Starts the specified jails or ALL.
+```
+
 ### Custom Release (e.g. running -CURRENT)
```
Comment (igalic) [inline README.md]: this would be nice to document
(also, how to convert a template to release ;)

**Pull request merged**